t/chaos/utils.go

```diff
@@ -158,16 +158,16 @@ func getPrometheusMetric(e *httpexpect.Expect, g *WithT, key string) string {
 	return targetSlice[1]
 }
 
-func getIngressBandwidthPerSecond(e *httpexpect.Expect, g *WithT) (float64, float64) {
-	key := "apisix_bandwidth{type=\"ingress\","
+func getEgressBandwidthPerSecond(e *httpexpect.Expect, g *WithT) (float64, float64) {
```

Comment:
Why even the semantics of this function was changed?

Reply:
> changed to egress bps comparison as measurement, since egress is what expected to be the same